specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkSecurityPerimeterList.json

```diff
@@ -13,7 +13,10 @@
             "name": "testNSP1",
             "id": "/subscriptions/subId/resourceGroup/rg1/providers/Microsoft.Network/networkSecurityPerimeters/testNSP1",
             "type": "Microsoft.Network/networkSecurityPerimeters",
+            "tags": {},
+            "location": "East US 2 EUAP",
             "properties": {
+              "perimeterGuid": "guid",
               "displayName": "TestNetworkSecurityPerimeter1",
```

Comment:
I think we need to remove displayName and description from NSP properties

Reply:
Done